<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shareDialog</class>
 <widget class="QDialog" name="shareDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>341</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Share with : </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="publicradioButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Share with Public</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="specificradioButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Share with specific user</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>580</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>580</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sharing File:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pathlabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>430</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
